### PR TITLE
Mini visual rework

### PR DESCRIPTION
**Pull request by gurrion**
Added pending mini visual rework

Mobile
localhost_8888_(Pixel 2 XL)
localhost_8888_(Pixel 2 XL) (1)
localhost_8888_(Pixel 2 XL) (2)

PC/Tablet
localhost_8888_(Laptop-1024x640)
localhost_8888_(Laptop-1024x640) (1)
localhost_8888_(Laptop-1024x640) (2)

### CODE REVIEW

File: pwa/build.sbt [hunk 1]
```diff
@@ -148,7 +148,7 @@ lazy val root = (project in file("."))
   .configure(baseSettings, browserProject, reactNpmDeps, withCssLoading, bundlerSettings, buildInfoSettings)
   .settings(
     useYarn := true,
-    webpackDevServerPort := 8080,
+    webpackDevServerPort := 8888,
```
Comment (AlexITC) [inline pwa/build.sbt:151]: Roll this back please

File: pwa/src/main/scala/net/wiringbits/cazadescuentos/API.scala [hunk 2]
```diff
@@ -11,11 +11,13 @@ case class API(
...
-  } else {
-    "http://localhost:9000"
-  }
+//  private val serverApi = if (BuildInfo.production) {
+//    "https://cazadescuentos.net/api"
+//  } else {
+//    "http://localhost:9000"
+//  }
+
+  private val serverApi = "https://cazadescuentos.net/api"
```
Comment (AlexITC) [inline pwa/src/main/scala/net/wiringbits/cazadescuentos/API.scala:20]: Roll this back please

**Review by AlexITC — APPROVED** [review]

File: pwa/src/main/scala/net/wiringbits/cazadescuentos/API.scala (hunk 2 above)
Comment (gurrion) [reply pwa/src/main/scala/net/wiringbits/cazadescuentos/API.scala]: done

File: pwa/build.sbt (hunk 1 above)
Comment (gurrion) [reply pwa/build.sbt]: done

**Pull request merged**